### PR TITLE
changes of visibility in documentation

### PR DESCRIPTION
**Pull request by senpl**
Changes in https://github.com/cypress-io/cypress/pull/29741 now updated in documentation.
This is still early so it should be done after this pull request date and version will be finally known.

### CODE REVIEW

File: docs/guides/core-concepts/interacting-with-elements.mdx [hunk 1]
```diff
@@ -59,10 +59,9 @@ Whenever Cypress cannot interact with an element, it could fail at any of the
 above steps. You will usually get an error explaining why the element was not
 found to be actionable.
 
-### Visibility
+### Hidability
 
-Cypress checks a lot of things to determine an element's visibility. The
-following calculations factor in CSS translations and transforms.
+Cypress checks a lot of things to determine if element is not hidden. The following calculations factor in CSS translations and transforms.
```
Comment (MikeMcC399) [inline docs/guides/core-concepts/interacting-with-elements.mdx:64]:
```suggestion
Cypress checks a lot of things to determine if an element is not hidden. The following calculations factor in CSS translations and transforms.
```

File: docs/guides/core-concepts/interacting-with-elements.mdx [hunk 2]
```diff
@@ -84,6 +83,19 @@ following calculations factor in CSS translations and transforms.
...
+
+Most of the time you should check if element is not hidden. Some elements might be visible but not clickable or interactable. 
+
+Cypress check visibility like browsers [checkVisibility](https://developer.mozilla.org/en-US/docs/Web/API/Element/checkVisibility).
+There are some exceptions from browser checks:
+- in `option` inside `select` visibility is determined by visibility of parent `select` element
+- if element is offscreen it is consided invisible
+- if parent cover element it is consided invisible
+In old browsers that not support [checkVisibility] check is hidden is performed.
+
```
Comment (MikeMcC399) [inline docs/guides/core-concepts/interacting-with-elements.mdx:98]: There are some grammatical and spelling errors in this section.

File: docs/guides/references/changelog.mdx [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
 ---
 
+## 13.14.0
+
+_Released 7/23/2024
+
+**Breaking Changes:**
+
+- Check of visibility is based on browsers checkVisibility. 'not.hidden' check is recommended before interaction with page elements now. 
+
```
Comment (MikeMcC399) [inline docs/guides/references/changelog.mdx:12]: - There should be no Changelog entry added here. The Changelog text should only be added through https://github.com/cypress-io/cypress/pull/29741

```suggestion
```

File: docs/guides/core-concepts/interacting-with-elements.mdx (hunk 2 above)
Comment (MikeMcC399) [reply docs/guides/core-concepts/interacting-with-elements.mdx:98]: I didn't comment in more detail because this PR is still in draft state.

**Comment by jennifer-shehane** [comment]
Closing as this is superceded by our 14 release docs

**Pull request closed without merging**